src/map/handler/Map.TouchZoom.js

```diff
@@ -109,6 +109,15 @@ L.Map.TouchZoom = L.Handler.extend({
 		    .off(document, 'touchmove', this._onTouchMove)
 		    .off(document, 'touchend', this._onTouchEnd);
 
+		var snap = L.Browser.any3d ? this._map.options.zoomSnap : 1;
+		if (snap) {
```

Comment:
Yes. One pull request per bug or feature. This pull request has **3** different things:
* A change in `L.Browser.chrome` that is not really needed (Edge having 'chrome' in the UA string doesn't break anything - if it breaks, send a bug report explaining what is broken) - and BTW this happens only in Edge 14, not in Edge 12
* A possible fix for #4592
* A change of behaviour in touch-zoom snapping (which has had no discussion at all, open up a bug about why it's needed)

As a general rule, don't make several changes in several pieces of the software in one commit, unless the changes are very tightly coupled (this is not such a case).